package.json

```diff
@@ -37,6 +37,7 @@
   },
   "dependencies": {
     "101": "^1.2.0",
+    "bluebird": "^3.3.5",
```

Comment:
It's just easier to implement caching using promises, that way failures are also cached (which can be considered good in some cases)